src/Stratis.Bitcoin.IntegrationTests/API/ApiSteps.cs

```diff
@@ -422,7 +422,7 @@ private void a_full_list_of_available_commands_is_returned()
         {
             var commands = JsonDataSerializer.Instance.Deserialize<List<RpcCommandModel>>(this.responseText);
 
-            commands.Count.Should().Be(20);
+            commands.Count.Should().BeGreaterOrEqualTo(22);
```

Comment:
I'm not a big fan of this test in general. Any time a new RPC method is added it breaks and I'm not sure what value it is really providing. I guess that all the rpc methods expected are available. Really there should be an integration test for each RPC method anyways that would fail if it wasn't there so I feel this might be redundant.

Reply:
hmm I am not sure here, probably keep the exact number else this test is a bit useless